gpMgmt/test/behave/mgmt_utils/gpactivatestandby.feature

```diff
@@ -76,6 +76,27 @@ Feature: gpactivatestandby
           And the tablespace is valid on the standby coordinator
           And clean up and revert back to original coordinator
 
+    Scenario: gprecoverseg recoveries failed segment when standby works instead of coordinator
+        Given the database is running
+          And all the segments are running
+          And the segments are synchronized
+          And the standby is not initialized
+         When the user initializes a standby on the host "sdw3" and port 7001
+         Then gpinitstandby should return a return code of 0
+          And verify the standby coordinator entries in catalog
+          And verify that pg_hba.conf file has "standby" entries in each segment (primary and mirror) data directories
+          And the coordinator goes down
+          And the user runs gpactivatestandby with options "-f"
+         Then gpactivatestandby should return a return code of 0
+          And verify the standby coordinator is now acting as coordinator
+          And user restarts from standby all primary processes for content 0
+         When the user runs command "gprecoverseg -aF" from standby coordinator
+         Then gprecoverseg should return a return code of 0
+         When the user runs command "gprecoverseg -r" from standby coordinator
+         Then gprecoverseg should return a return code of 0
+          And the standby coordinator goes down
+          And revert back to original coordinator
+
```

Comment:
why not `clean up and revert back to original coordinator`?

Reply:
it was before last commit

Reply:
It does not work, because standby works in another host.

Reply:
Strange. According to the code, it should work with another host. https://github.com/arenadata/gpdb/blob/dc3b23a2b6e513ebc12610965470ef5b6484364c/gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py#L1294-L1312